prov/sm2/src/sm2_rma.h

```diff
@@ -0,0 +1,34 @@
+#include "sm2.h"
```

Comment:
Missing copyright.
Also I don't think this separate file is really necessary? All of these can just be static inside rma.c/progress.c

Reply:
ack on copyright thanks.

I actually would rather have all of these be in a sm2_sar.c/h file and have progress jump into various protocols, but I don't want to refactor current organization right now.

Would you prefer this file be named sm2_sar.h?

Reply:
I'm ok with having a separate header but it really needs to be isolated to what it does (whether SAR or RMA). It looks like this does both which is why it's confusing.

Reply:
Removed the rma.h header and renamed several of those functions to be sar specific rather than rma specific.  You can find them in sm2.h now.